src/store/undoRedo/index.ts

```diff
@@ -165,7 +165,12 @@ export const actions: ActionTree<IUndoRedoState, IRootState> = {
 
     // ####### force patch ##############
     if (isNewSnapshotFromPersistenceAction) {
-      commit('patchSnapshot', newSnapshot)
+      if (getters.hasCurrentSnapshot as boolean) {
+        commit('patchSnapshot', newSnapshot)
+      } else {
+        // We cannot patch non-existing snapshots
+        commit('addSnapshot', newSnapshot)
```

Comment:
This case comes from this condition; I wondered when this may happen?

https://github.com/FLOIP/flow-builder/blob/b48cfff3dafbca86a893786c293aff4cbdabf45f/src/store/undoRedo/index.ts#L135-L139